bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ZVG_4f_NLO/ZVG_4f_NLO_proc_card.dat

```diff
@@ -0,0 +1,6 @@
+import model loop_sm
+define lep+ = e+ mu+ ta+
+define lep- = e- mu- ta-
+define v = z w+ w-
+generate p p > v lep+ lep- a $$ h [QCD]
```

Comment:
Hi Meng, does this not have same issues that Olivier explained in [1]?

[1] https://answers.launchpad.net/mg5amcnlo/+question/696694

Reply:
Hi Sihyun, the semi-leptonic channel will not be affected by the ambiguity, only the identical decay of two Z will be affected,  there is the answer from Olivier, https://answers.launchpad.net/mg5amcnlo/+question/696360

Reply:
Hmmm it's now more puzzling.

The cross section that you do with both a. and b. would be
sigma(p p > z lep+ lep-) that you get from MadGraph TIMES BR(Z > q q OR lep+ lep-) in MadSpin

a. MG : p p > z lep+ lep- & MS : z > q q
b. MG : p p > z lep+ lep- & MS : z > lep+ lep-

Then you mean only for d., the cross sections will be somewhat buggy? And c. will be OK?
c. MG : p p > z e+ e- & MS : z > mu+ mu-
d. MG : p p > z e+ e- & MS : z > e+ e-

Reply:
Hi Sihyun, from Olivier's answer, the issue is from the ambuiguity of same lepton pairs, if muon is different from ele in the module, then the 'c' should be ok. But in the semileptonic case, as Olivier said, there should be no ambuiguity as there are quark and lepton pair.

Reply:
I don't fully get this issue frankly.
a. p p > z e+ e-, z > mu+ mu-
b. p p > z e+ e-, z > e+ e-
The cross sections of a. and b. are exactly the same when I give mmll=10 GeV
Is this expected? Sorry for my ignorance :|

Reply:
i think it's expected. if only consider two onshell Z, the phase space of final state e+e-e+e- should be half of the e+e-mu+mu-. As in 4e case, "p p > z e+ e-, z > e+ e-" is buggy, in which two onshell Z is counted twice, which make it agree with the correct syntax "p p > z e+ e-, z > mu+ mu-". i made some runs to confirm it:

all with "mmll=10", and with "True = cut_decay" if decay chain
syn1: p p > e+ e- e+ e-  XS: 0.005757 pb
syn2: p p > e+ e- z, z > e+ e- XS: 0.009628 pb (buggy one)
syn3: p p > z z, z > e+ e- XS: 0.004405 pb
syn4: p p > e+ e- e+ e- $z  XS: 0.0001746 pb
--> syn1 ~ syn2+syn4-syn3

syn5: p p > e+ e- mu+ mu- XS: 0.01151 pb
syn6: p p > e+ e- z, z > mu+ mu- XS: 0.009643 pb
syn7: p p > mu+ mu- z $z, z > e+ e- XS: 0.0008272 pb
syn8: p p > e+ e- mu+ mu- $z XS: 0.0002583 pb
--> syn5 ~ syn6+syn7+syn8

Reply:
Ahhhh

Ok so the problem comes from the fact that when you have two Z bosons that decays into ee and mumu, you can distinguish the two. But for ee and ee case, there's no way to distinguish it because they are identical

Reply:
Just out of curiosity, is there a sample that only include the higgs contributions? Or is the cross section negligible?

Reply:
i did a test on following two processes (l=e,mu):
1. p p > z l+ l- a $$h [QCD]
2. p p > z l+ l- a [QCD]
with default cut setting, the xs are the same 1.855e-02 +- 5.3e-05 pb